docs/upgrading.md

```diff
@@ -0,0 +1,73 @@
+# Upgrading the Agent
+
+This document provides user guidance around agent upgrades and versioning.
+Like every other piece of software included in your project, the agent 
+should be upgraded frequently enough to pick up ongoing enhancements 
+while allowing room for testing. 
+
+## Watching For New Releases
+
+How can you find out about new releases? The best way is to "Watch" this repo
+from the "Watch" button in the upper-right corner of the main repository 
+page. If you don't want to be notified for every pull request and issue, you can 
+choose "Custom" under "Watch" and select just the "Releases" checkbox.
+
+Another approach is to just check in regularly with the GitHub 
+[releases page](https://github.com/signalfx/splunk-otel-java/releases) for this 
+project. 
+
+## When Does Splunk Release?
```

Comment:
"When does Observability Cloud release?"

Note: The Splunk documentation style guide has this comment regarding the use of "Splunk":

When used alone, use "Splunk" to refer only to the company, not to any product. Don't use in the possessive form.

Reply:
I don't think it's supposed to mean the entire Observability Cloud, it's just this project:
```suggestion
## When does Splunk Distribution of OpenTelemetry Java release?
```

Reply:
👍🏻